### DISCUSSION TITLE
Integrating epic-stack with vanilla-extract

### DISCUSSION THREAD

**Discussion by MoSattler** [epicweb-dev/epic-stack#406] [Q&A] [answered]
**Hello,**

Has anyone successfully integrated the epic-stack with vanilla-extract as described in the [remix docs](https://remix.run/docs/en/main/guides/styling#vanilla-extract)?

Facing this issue:
```typescript
import { button } from "./styles.css";
```
Error:
```plaintext
Module '"*.css"' has no exported member 'button'. Did you mean to use 'import button from "*.css"' instead?ts(2614)
```

Any suggestions would be appreciated. Thanks.

**Comment by kentcdodds** [comment, marked as answer]
Hi!

You'll need to add the .ts file extension.

**Reply by MoSattler** [reply]
Hi @kentcdodds ,

Thank you for the quick answer.

I double checked the [remix docs](https://remix.run/docs/en/main/guides/styling#vanilla-extract) regarding Vanilla Extract. It explicitly states to omit the .ts during imports:
```typescript
import * as styles from "./styles.css"; // Note that `.ts` is omitted here
```
Given this, I just want to double-check: is adding the .ts extension indeed the recommended solution?

**Reply by kentcdodds** [reply]
Yeah, in this repo we're using native ESM which requires the extension. They only have the comment there because otherwise people may be confused and think they were importing a CSS file
